### PR TITLE
Fixed the required changes assinged 

### PR DESCRIPTION
**Pull request by Fab-Gaurav**
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
I have fixed the changes assigned : https://github.com/TesseractCoding/NeoAlgo/commit/5c83a912eb427872aa886fc32ec49eddffb273f4
Issue #4096

### CODE REVIEW

File: C-Plus-Plus/math/Unique_Number.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+the elements that appears exactly once in the array. Return the sum of all 
+Unique elements of nums.
+
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+
+
```
Comment (HarshCasper) [inline C-Plus-Plus/math/Unique_Number.cpp:13]: Remove extra lines of code

File: C-Plus-Plus/math/Unique_Number.cpp [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+/*
+
+INPUT : [1, 5, 2, 6, 5]
+OUTPUT : 9
+
+Explanation : Unique elements are [1, 2, 6] and the Sum is 9
+
+*/
```
Comment (HarshCasper) [inline C-Plus-Plus/math/Unique_Number.cpp:63]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Format and indent the Code

**Review by ankitaggarwal23 — COMMENTED** [review]
@Fab-Gaurav Don't open new PR for each changes requested by @HarshCasper. Closing all your PR open a new one Do all the changes Harsh requested.

**Pull request closed without merging**